mesos_slave/assets/service_checks.json

```diff
@@ -13,5 +13,20 @@
             "url"
         ],
         "description": "Returns CRITICAL if the Agent cannot connect to the Mesos slave metrics endpoint, otherwise OK."
+    },
+    {
+        "agent_version": "5.2.0",
+        "integration": "Mesos",
+        "check": "mesos.can_connect",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "name": "Can Connect",
+        "groups": [
+            "host",
+            "url"
+        ],
+        "description": "[Deprecated] Returns status after pinging the Mesos instance url. Additional information about response status at the time of collection is included in the check message."
```

Comment:
This service check is not sent but was documented in the previous `mesos` integration definition.
The goal is to migrate it here first for the tile split and then delete it once the split is good